.circleci/docker/build.sh

```diff
@@ -82,23 +82,23 @@ case "$image" in
     GCC_VERSION=7
     # Do not install PROTOBUF, DB, and VISION as a test
     ;;
-  pytorch-linux-xenial-py3.6-gcc5.4)
-    ANACONDA_PYTHON_VERSION=3.6
+  pytorch-linux-xenial-py3.7-gcc5.4)
```

Comment:
Do we need these particular changes for deprecating 3.6 binary builds? I think these are just for regular CI builds

Reply:
Hm, what do you suggest ? We get rid of these jobs completely ?